frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -455,11 +456,13 @@ export class HtmlDocument extends React.Component {
     }))
 
     removeLocalStorageItem(
-      this.state.appName,
-      this.state.content.content_id,
-      this.state.content.workspace_id,
+      state.appName,
+      state.content.content_id,
+      state.content.workspace_id,
       LOCAL_STORAGE_FIELD.RAW_CONTENT
     )
+
+    window.history.replaceState(null, '', PAGE.WORKSPACE.CONTENT(state.content.workspace_id, state.content.content_type, state.content.content_id))
```

Comment:
why using a .replaceState?

Reply:
To update the url without reloading the page (toggle between the edition/not edition mode)

Reply:
can't you use the react router object to do so?

Reply:
I didn't find a way to do it

(note : ref about using replaceState https://stackoverflow.com/questions/57101831/react-router-how-do-i-update-the-url-without-causing-a-navigation-reload)

Reply:
do we need to change the url then?
I mean, if I copy the url changed by the `replaceState`, will I end up in the right state?

Reply:
yes, to handle the mode on Notes, a query param is used temporarily on the URL during edition. If you copy the URL before replace you are in edition mode, and after replace you are in view mode.

Reply:
@Skylsmoi

Reply:
mmh, then shouldn't the new url be triggering the mode change instead of doing it programmatically and then changing the url by forcing a non refresh of the component?